data/virtualization/autoyast/host_12.xml.ep

```diff
@@ -69,6 +69,9 @@
         % if ($key eq 'ltss') {
         <reg_code><%= $get_var->('SCC_REGCODE_LTSS') %></reg_code>
         % }
+        % if ($key eq 'ltss_es') {
+        <reg_code><%= $get_var->('SCC_REGCODE_LTSS_ES') %></reg_code>
```

Comment:
This will add handling requirement in this task -- adopt MU CI tool A5, https://progress.opensuse.org/issues/160868. I will extend it later.